Editor/Resources/EditorWindow/GameLiftPlugin.uss

```diff
@@ -45,9 +49,10 @@
 
 .tab__content {
     display: none;
+    padding: var(--spacing-large);
+    min-width: var(--content-width);
```

Comment:
Consider renaming `--content-width` to `--page-width` or something else specific since `content` is too vague for a generic variable (can refer to tab__content, page__content, or another content we may add later)

Reply:
renamed